how-not-to-be-a-dinosaur.md

```diff
@@ -0,0 +1,135 @@
+
+Then there was news: [React Hooks](https://reactjs.org/docs/hooks-overview.html#%EF%B8%8F-effect-hook) was about to be released in React 16.8. This made code _so_ much more readable that I started thinking "Hmm, maybe I can do this".
+
+I don't want to go all geeky on the differences between hooks and classes and why hooks are so awesome, especially since there are [excellent](https://reactjs.org/docs/hooks-intro.html) articles [written](https://itnext.io/whats-the-big-deal-with-react-hooks-d73145eb14e0) on this [subject](https://medium.com/frontmen/react-hooks-why-and-how-e4d2a5f0347). If you like to learn more about this, you should check [them](https://reactjs.org/docs/hooks-overview.html) out.
+
+For me personally the biggest thing was the improved simplicity of function components in hooks. Where it felt like jumping to hoops to keep track of some kind of state, hooks just let me use [`useState()`](https://reactjs.org/docs/hooks-state.html).
+
+For example, in oldskool classes, I used to write this to increment a counter:
+```javascript
+this.setState({ count: this.state.count + 1 })
+```
+
+Where hooks let me do this:
+
+```javascript
+setCount(count + 1)
+```
+I bet that if you are reading this you'll understand it even if you have little (react) programming experience.
+
+After this "challenge accepted!" point the hardest obstacle had to be overwon: how to handle the "I have to start all over again" feeling. I felt like I knew nothing. I was Jon Snow.
+
+## Dealin' with dem feelins...
+
+Who had I been kidding all these years calling myself a senior developer? How embarrassing was it that I had to ask my 12 years younger colleague for help over and over again...  
+
+![](https://raw.githubusercontent.com/MyOnlineStore/blogs/Dinosaurusblog/public/johan_knowsnothing.png)
+
+Some people will think "what's the big deal?", and I am happy for you if you do, but you have to understand that asking for help was a big issue for me. And a major cause for my burn out and the personality disorder that came with it.
+
+So I did two things: I decided to start learning all these new tools and frameworks and more importantly I started to ask for help. But the most important thing I did: I told my colleagues how I was feeling and how insecure I was about everything. They were very supportive and this gave me the confidence to go for it.
+
+One of the most important things they gave me is that they showed me that I did make progress. A lot, actually. And that it's not the end of the world not knowing everything. As long as I kept learning, things will work out in the end, was the message I got.
+
+Also, I realised I did not have to start all over again. It might have felt like it, but I still had a big headstart with my years of experience in programming overall. It was just a new framework and new tools that I needed to learn, not learning how to speak and not the first tool I ever used.
```

Comment:
```suggestion
Soon, I also realized I did not have to start all over again at all. It might have felt like it, but I still had a big headstart with my years of experience in programming overall. It was just a new framework and new tools that I needed to learn, not learning how to speak and not the first tool I ever used.
```